schemas.md

```diff
@@ -43,6 +43,19 @@ The user's geolocation at that time.
 - `description`: Text description, such as messages sent by a GPS beacon.
 - `mixin:location`
 
+### activity.motion
```

Comment:
I think that different schemas within the same namespace would make more sense here: `activity.exercise.running`, `activity.exercise.cycling`, ...

The idea is that each of these activities have shared attributes (calories, duration), but also differing ones (step count).

Reply:
That makes sense, but there are a lot of possible activities (see [google documentation](https://developers.google.com/fit/rest/v1/reference/activity-types), looks around 100). I've only tested for the 3-4 that I have any record of. (running, cycling, walking, rowing). 
Would you suggest adding to the schema only those that are explicitly handled in the code, and have a left over "generic" exercise for the remainder?